lib/ramble/ramble/workspace/workspace.py

```diff
@@ -935,6 +1065,7 @@ def yaml_add_comment_before_key(
 
             return base
 
+        from ramble.util.conversions import list_str_to_list
```

Comment:
Should this import be higher?

Reply:
In this method? Or more like a module level import?

I only imported it here, because it's a kind of weird utility that nothing else needs (kind of like the yaml import below it).

But I could move it up to module level too.